services/order.py

```diff
@@ -0,0 +1,36 @@
+import datetime
+
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, MovieSession
+
+
+def create_order(tickets: list[dict], username: str,
+                 date: datetime.datetime = None) -> None:
+    user = get_user_model().objects.get(username=username)
+
+    with transaction.atomic():
```

Comment:
use `@transaction.atomic` instead of with 
this will make your code more readable and clean